prac_05/word_occurrences.py

```diff
@@ -0,0 +1,21 @@
+def main():
+    count_word_occurences = {}
+    user_string = []
+    user_string = input("Please enter a string")
+    split_user_string = user_string.split()
+    split_user_string.sort()
+    print(split_user_string)
+    longest_len_word = 0
+    for i in split_user_string:
+        if len(i) > longest_len_word:
+            longest_len_word = len(i)
+        if i in count_word_occurences:
+            count_word_occurences[i] += 1
+        else:
+            count_word_occurences[i] = 1
```

Comment:
there is a short way to count word_length 
eg.
longest_word = max(len(word) for word in words)

Reply:
feedback implemented https://github.com/IanEllimanJCU/cp1404practicals/commit/fbe867decf4ffc690cd2984bb3f3c327c0e5f65f